### PR TITLE
Make it clear that wallet is best source of greylist information

### PR DESCRIPTION
**Pull request by makeasnek**
And provide instructions on how to find it in the wallet

### CODE REVIEW

File: guides/whitelist.htm [hunk 1]
```diff
@@ -22,7 +22,6 @@ <h5>Please note:</h5>
                     <p>Any BOINC work done outside of the whitelisted BOINC projects will not be rewarded Gridcoin.</p>
                     <p>Want a project whitelisted? Ask about it on the <code>#whitelisting-committee</code> channel on the Gridcoin Discord! More details about adding a project to the whitelist can be found on the <a href="/wiki/whitelist-process.html">whitelist process wiki page</a></p>
                     <p>You can check whitelisted project details on <a href="https://gridcoinstats.eu/project">Gridcoinstats</a>.</p>
-                    <p>The whitelist is verifiable on the wallet on the projects tab on the beacon menu or with the <code>listprojects</code> command</p>
```
Comment (barton2526) [inline guides/whitelist.htm:25]: why is this being removed? It is accurate

File: guides/whitelist.htm [hunk 2]
```diff
@@ -32,7 +31,7 @@ <h5>Please note:</h5>
                 <div class='col-12 text-center'>
                     <h3>Greylist</h3>
                     <p>When projects are running out of work or are offline for temporary maintenance they are placed on the Greylist.  These projects will not reward Gridcoin until they meet the requirements to be placed back on the whitelist again.  Please see various social media (Twitter, Facebook, Reddit, Slack) for notices about greylisted projects.</p>
-                    <p>Please see <a href="https://gridcoin.ddns.net/pages/project-list.php">this unofficial Gridcoin Greylist monitoring website</a> for daily updates on which projects are added and removed from the Greylist.</p>
+                    <p>Please see <a href="https://gridcoin.ddns.net/pages/project-list.php">this unofficial Gridcoin Greylist monitoring website</a> for daily updates on which projects are added and removed from the Greylist. </p>
```
Comment (barton2526) [inline guides/whitelist.htm:34]: Honestly, this line should probably be removed entirely. The greylist site by G_UK may never return

File: guides/whitelist.htm (hunk 1 above)
Comment (makeasnek) [reply guides/whitelist.htm:25]: Because I moved it down further on the page

File: guides/whitelist.htm (hunk 2 above)
Comment (makeasnek) [reply guides/whitelist.htm]: Removed.

**Review by barton2526 — APPROVED** [review]

**Pull request merged**